test/unit/io.js

```diff
@@ -735,6 +779,12 @@ describe('Input/output', function () {
     });
   });
 
+  it('Invalid WebP alpha quality throws error', function () {
```

Comment:
Please can you add tests for invalid `lossless` and `nearLossless` attributes also.